### DISCUSSION TITLE
Working Owncast in windows

### DISCUSSION THREAD

**Discussion by jakubix30** [owncast/owncast#4061] [Ideas]
hi, I made a working modification/fix for owncast to make the server work on windows, for now when I tested everything works, I want to share what I did and I hope that developers will add official support for windows in the future ;)
Here is my repository https://github.com/jakubix30/Owncast-windows-fix

**Comment by gabek** [comment]
If this is something you'd like to pursue, go ahead and open an issue for it and we can talk about logistics. You can put up your PR pointing to that issue.
